pkg/realmd/operation.js

```diff
@@ -133,7 +135,7 @@ function instance(realmd, mode, realm, button) {
                 $(".realms-op-address-spinner").hide();
         });
 
-        realmd.call(MANAGER, PROVIDER, "Discover", [ name, { } ])
+        realmd.call(MANAGER, PROVIDER, "Discover", [ name, { } ], TIMEOUT)
```

Comment:
How is this related to fixing the error message? It seems to work fine here without the timeout. We'd usually expect realmd to deliver a positive or negative result fast, unless realmd itself is broken.

Not too long ago I had a rather long discussion with @stefwalter about timeouts. Personally I also favor having timeouts on everything, but it seems cockpit traditionally hasn't used them, so that it works on heavily loaded machines.

I'd prefer reverting the timeout bit here.

Reply:
Thanks for shedding light on the timeout stuff. Removing the timeout from the call.